release/pipelines/windows-efi-installer/windows-efi-installer.yaml

```diff
@@ -61,6 +68,34 @@ spec:
       name: isoDVName
       type: string
   tasks:
+    - name: import-autounattend-configmaps
+      params:
+        - name: SCRIPT
+          value: |-
+            if [ $(params.acceptEula) ]; then
+              curl -o configmap.yaml $(params.autounattendXMLConfigmapsURL)
+              sed -i "s/<!-- <AcceptEula>true<\/AcceptEula> -->/<AcceptEula>true<\/AcceptEula>/g" configmap.yaml
```

Comment:
This should look for false and replace with true?

Reply:
this behaviour will be changed in https://github.com/kubevirt/kubevirt-tekton-tasks/pull/410